app/assets/javascripts/va_facilities/facility_adopted_practice_search.es6

```diff
@@ -1,79 +1,90 @@
+const ap = {
```

Comment:
⚠️  had to wrap these values in an object, since when we load the javascript any previously loaded javascript's constants are carried over since I think the JS gets all loaded together